sphinx/screens/dashboard/dashboard/src/main/res/layout/layout_dashboard_chats.xml

```diff
@@ -6,11 +6,17 @@
     android:layout_height="match_parent"
     android:background="@color/body">
 
-    <androidx.recyclerview.widget.RecyclerView
-        android:id="@+id/recycler_view_chats"
+    <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/swipeContainer"
```

Comment:
Rename id to follow the pattern we are using all over the project: It could be: "layout_swipe_refresh_chats"